commerce-theme-breccia/src/css/components/_widget.scss

```diff
@@ -0,0 +1,96 @@
+.widget-mode-detail {
+	background-color: #FFF;
+
+	.title {
```

Comment:
These nested selectors should be sorted alphabetically.

Same throughout the file.